join.sh

```diff
@@ -70,3 +70,5 @@ docker run -t --rm \
         join \
             --hostname $(hostname) \
             --config-path /target$(realpath $1)
+
+touch /var/lib/prom.done
```

Comment:
The short-term plan is that Drydock will place a systemd unit that launches on boot, which first checks for this file to see whether it should run join or not.  If the file is there, then it exits, otherwise it runs join.